client/src/components/Tabs/TabItem.js

```diff
@@ -13,39 +13,32 @@ class TabItem extends Component {
       className,
       extraClass,
       disabled,
-      bsClass,
-      onEnter,
-      onEntering,
-      onEntered,
-      onExit,
-      onExiting,
-      onExited,
-      animation,
-      unmountOnExit,
       } = this.props;
 
     return {
-      eventKey: name,
+      tabId: name,
       className: `${className} ${extraClass}`,
       disabled,
-      bsClass,
-      onEnter,
-      onEntering,
-      onEntered,
-      onExit,
-      onExiting,
-      onExited,
-      animation,
-      unmountOnExit,
     };
   }
 
+  /**
+   * Checks if the tab is the active tab
+   *
+   * @returns {bool} active
+   */
+  isActive() {
+    return this.context.activeTabId === this.props.name;
+  }
+
   render() {
     const tabProps = this.getTabProps();
     return (
-      <Tab.Pane {...tabProps}>
-        {this.props.children}
-      </Tab.Pane>
+      <TabPane {...tabProps}>
+        <Fade in={this.isActive()} unmountOnExit>
```

Comment:
I don't think unmountOnExit is a good idea, at least in ss4 context.

The input values and markup will be removed from the HTML, which is fine if you have the values recorded in something like redux-form... but if the field was used outside this context - e.g. an entwine section which we will need to support, then those values will not be posted back until the tab was shown which means a different set of values will be lost instead.